### ISSUE TITLE
How to use in form ?

### ISSUE THREAD

**Issue by khallid-itdevtech** [prrashi/rateYo#70]
is it work with form ?

**Comment by bozlurrahman** [comment]
Yes, It works with form, follow the link [https://rateyo.fundoocode.ninja/#option-onSet](https://rateyo.fundoocode.ninja/#option-onSet) to set and get rating input.
Thanks.
